public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
Must these be changed?

Reply:
These seem to have no effect on the website, but I think it's better to change it. It provides the information of a web app.